src/features/index.ts

```diff
@@ -8,6 +8,16 @@ import { FeatureManager } from './featureManager';
 
 export type { FeatureManager } from './featureManager';
 
+class DashboardContextMenuManager extends FeatureManager {
+    type = UiAppFeatureType.DASHBOARD_CONTEXT_MENU;
+    eventsToSubscribe = [UiAppEventToSubscribeType.DASHBOARD_CONTEXT_MENU];
```

Comment:
What does this event do?

Reply:
Before the recent changes, it would inform the 3p app when the context menu is triggered -- this may be outdated now.

Reply:
Another note: this event doesn't send any new information to the 3p app as it is still TBD what type of context would be useful.

Reply:
thanks for the context!